### PR TITLE
Initial CPS Takeover

### PR DESCRIPTION
**Pull request by kevinconaway**
Changes in this PR:

- Initial refactor to use maven instead of gradle and remove all code unrelated to Graphite.  
- Update the dropwizard metrics library version to take advantage of bug fixes to the socket handling

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.walmart.cps</groupId>
```
Comment (rugden) [inline pom.xml:5]: `com.walmart.cps` cool!

**Review by rugden — APPROVED** [review]

**Review by rahmantasnim — APPROVED** [review]

**Review by ssinganamalla — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                    <source>1.7</source>
```
Comment (woshizhoutong) [inline pom.xml:114]: why not 1.8?

Comment (kevinconaway) [reply pom.xml:114]: The existing library targeted 1.7 so I left it as is. 

I have mixed feelings. On the one hand, java7 is 3 years eol by now. On the other, a lot of orgs still use it and its not very burdensome to target it here 

Thoughts?

Comment (rahmantasnim) [reply pom.xml:114]: I'm on the side of keeping it compatible with java 7, especially bc it doesn't really take away much.

**Pull request merged**